File: oddEven.cpp
#include <iostream>
using namespace std;

bool isEven(int n){
    //finding reminder of n
    int rem = n%2;
    if(rem ==0){
        return true;
    }
    else{
        return false;
    }

}

int main(){
    int n=16;
    if(isEven(n))
        cout << "true";
    else
        cout << "false";
    
    return 0;
}

File: oddEven.py
def isEven(n):

# finding reminder of n
    rem = n % 2;
    if rem == 0:
        return True
    else:
        return False


if __name__ == "__main__":
    n=45
    if isEven(n):
        print("true")
    else:
        print("false")